hcp/io/read.py

```diff
@@ -265,20 +265,37 @@ def _hcp_pick_info(info, ch_names):
 
 
 def read_trial_info_hcp(subject, data_type, run_index=0, hcp_path=op.curdir):
-    """ read trial info """
+    """ read trial info
+
+    Parameters
+    ----------
+    subject : str
+        The HCP subject.
+    data_type : str
```

Comment:
Yes, this is why it's wip, need to look it up and apply index correction. This is why it's wip.